src/libraries/Common/src/System/Security/Cryptography/PqcBlobHelpers.cs

```diff
@@ -216,7 +216,7 @@ internal delegate TReturn EncodeMLKemBlobCallback<TState, TReturn>(
             string blobKind,
             ReadOnlySpan<byte> blob);
 
-        internal static TReturn EncodeMLKemBlob<TState, TReturn>(
+        internal static unsafe TReturn EncodeMLKemBlob<TState, TReturn>(
```

Comment:
Consider using an unsafe block instead of the unsafe keyword on the method, see https://github.com/agocke/designs/blob/caller-unsafe/proposed/caller-unsafe.md.